docs/monitoring/manually-setting-jmx-containers.md

```diff
@@ -0,0 +1,67 @@
+title: Manual set up of JMX monitoring for Java process running in containers
+description: In some cases it may be required to set up JMX settings of monitored process manually
```

Comment:
description: In some cases it may be required to set up JMX settings of monitored process manually
=>
description: In some cases, it may be required to set up JMX settings for monitored processes manually. This explains the procedure of setting it up manually.